torch/_tensor_docs.py

```diff
@@ -1170,6 +1170,45 @@ def add_docstr_all(method, docstr):
 See :func:`torch.index_select`
 """)
 
+add_docstr_all('sparse_mask',
+               r"""
+sparse_mask(input, mask) -> Tensor
+
+Returns a new SparseTensor with values from Tensor :attr:`input` filtered
+by indices of SparseTensor :attr:`mask`. :attr:`input` and :attr:`mask`
+must have the same shape.
```

Comment:
nit: You could be even more explicit here if you said, values of mask are ignored.

Reply:
Do you mean: `values` of `mask` SparseTensor is not used? Surely yes I can include that.